### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by ashcrow**
A few test case additions.

### CODE REVIEW

File: test/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2016  Red Hat, Inc
```
Comment (mbarnes) [inline test/test_cli.py]: We talked after the new year about updating the copyrights.  This is why I'd kinda prefer doing them all at once.

**Review by mbarnes — COMMENTED** [review]
Tests look okay, though Travis is whinging about some `bytes` vs `str` issue in Python 3.  I think I've seen this before and it doesn't appear to be coming from your changes, at least not directly.  Not sure if it's a Python 3.3 vs 3.5 thing.  I'll look into it a little more before approving.

**Comment by ashcrow** [comment]
I was able to reproduce the errors after updating libraries. Will update.

**Comment by ashcrow** [comment]
The failures were not related to my changes, but I included a fix along with a copyright year updates.

⬆️

**Comment by ashcrow** [comment]
I can not reproduce the last error:

```
nose.proxy.AttributeError: 'NoneType' object has no attribute '__context__'
```

This looks like it's something travis specific. I updated to the latest everything and still can't get it to occur.

**Comment by ashcrow** [comment]
It was travis specific. Forced upgrade on components and nose works properly.

**Comment by ashcrow** [comment]
I'm going to try pulling tox to control the environment and see what happens.

**Comment by ashcrow** [comment]
Updated version of python used in the tests.

**Review by mbarnes — APPROVED** [review]
Nice, thanks!

**Pull request merged**